Frontend/src/components/pages/Company.jsx

```diff
@@ -567,34 +244,34 @@ export default function Company() {
               <AccordionDetails>
                 <List dense>
                   <ListItem disablePadding>
-                    <ListItemText primary={"Domain: " + companyInfo.domain} />
+                    <ListItemText primary={"Domain: " + company.domain} />
```

Comment:
Ovdje treba umjesto ,domain staviti .sector (ovo je Ana promijenila na sljedecem PR-u pa cim mergamo to ce raditi dobro)

Reply:
U tom slučaju prvo mergajte Anino. Općenito vrijedi da ako ima nekih problema s mergevima, da onaj koji zadnji merga i ispravlja. Kada bi sada ispravio to, bez ANinog koda, ne bi mi radilo, pa mi nema baš smisla.